backend/src/GraphQl/AccessRights/DataAccessRightsMutations.cs

```diff
@@ -0,0 +1,212 @@
+﻿using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Database.Data;
+using Database.Services;
+using HotChocolate.Types;
+using Microsoft.AspNetCore.Http;
+using Microsoft.EntityFrameworkCore;
+
+namespace Database.GraphQl.AccessRights;
+
+[ExtendObjectType(nameof(Mutation))]
+public class DataAccessRightsMutations
+{
+    public async Task<AddDataAccessRightsPayload> AddAccessRightsToDataAsync(
```

Comment:
Please call the mutation `AddDataAccessRightsAsync` to make it consistent with the naming of the payload and the naming of the `DataAccessRights` model itself.